docs/components/links.md

```diff
@@ -4,7 +4,21 @@ layout: variation
 section: components
 secondary_section: Text
 status: Released
-description: "Links lead users to a different page or further information. In contrast, buttons are used to signal actions. Users should be able to identify links without relying on color or styling alone.\n\nMore information can be found at:\n* http://cfpb.github.io/design-manual/page-components/links.html\t\n* https://cfpb.github.io/capital-framework/components/cf-core/#default-links\n* https://cfpb.github.io/capital-framework/components/cf-typography/#link-patterns\""
+description: >-
+  Links lead users to a different page or further information. In contrast,
+  buttons are used to signal actions. Users should be able to identify links
+  without relying on color or styling alone.
+
+
+  More information can be found at:
+
+
+  * <https://cfpb.github.io/design-manual/page-components/links.html>
+
+  * <https://cfpb.github.io/capital-framework/components/cf-core/#default-links>
+
+  *
+  <https://cfpb.github.io/capital-framework/components/cf-typography/#link-patterns>
```

Comment:
We shouldn't link to capital framework or design manual URLs any more, since they are to be superseded by the design system documentation, but this is fine for now.